### DISCUSSION TITLE
Sending "Request status of link"

### DISCUSSION THREAD

**Discussion by xiejianulm** [community/community#79078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For IEC 60870-5-101: Balanced, 1 link byte, Slave:

What is the correct seting or sending the "Request status of link" (10 49 01 4A 16) at the begine of  initial the connection?

Thank you for the help

**Comment by queenofcorgis** [comment]
Hi @xiejianulm ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we will close this duplicate discussion, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
